### PR TITLE
Add the logo as icon and background

### LINKED ISSUE #165: Add the logo as icon and background
### Feature you want to implement
The logo will be drawn in `self.centralWidget`.

### How the feature is implemented
- The background image is stored in `resources/background.png`.
- The implementation is referred to the [artiq dashboard](https://github.com/m-labs/artiq/blob/master/artiq/frontend/artiq_dashboard.py#L82).

### Additional context
Thanks for designing this "QIWIS" logo. @choiwh03 

<image width="30%" src="https://github.com/snu-quiqcl/qiwis/assets/76851886/6fd55f62-3224-4327-928c-fb66fb548e0b">

### PR DESCRIPTION
**Pull request by BECATRUE**
This PR will close #165.
For details, please read the contents of #165.

# Summary
- Both icon and background image paths are set in `config.json`.
- The background image is drawn in `self.centralWidget`, an instance of `MdiArea`.

# Screenshot
image

### CODE REVIEW

File: qiwis.py [hunk 1]
```diff
@@ -140,6 +141,34 @@ class QiwiscallResult(Serializable):
...
+        background: The QPixmap instance of the background image.
+    """
+
+    def __init__(self, background_path: Optional[str] = None):
+        """Extended.
+        
+        Args:
+            background_path: The path of the background image.
+        """
+        QMdiArea.__init__(self)
```
Comment (kangz12345) [inline qiwis.py]: Why don't you use `super().__init__()`?

File: qiwis.py [hunk 2]
```diff
@@ -157,6 +186,7 @@ class Qiwis(QObject):
     def __init__(
         self,
         appInfos: Optional[Mapping[str, AppInfo]] = None,
+        constants: Optional[Tuple] = None,
```
Comment (kangz12345) [inline qiwis.py:189]: Please write the docstring!

**Review by kangz12345 — CHANGES_REQUESTED** [review]

File: qiwis.py (hunk 1 above)
Comment (BECATRUE) [reply qiwis.py]: I couldn't understand it well, but it makes an error.
Also, the ChatGPT also told me the code above, so I did so.

Comment (kangz12345) [reply qiwis.py]: Could you explain in more detail? What errors did you get? What did ChatGPT say?

Comment (BECATRUE) [reply qiwis.py]: Oh, I'm really sorry that I was mistaken.
`super().__init__()` works well and I will apply it.

**Review by kangz12345 — APPROVED** [review]
LGTM

**Pull request merged**